assets/src/stories-editor/plugins/context-menu-item.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Plugin for add remove page button to more settings.
+ */
+
+/**
+ * External dependencies
+ */
+import { ReactElement } from 'react';
+/**
+ * WordPress dependencies
+ */
+import { PluginBlockSettingsMenuItem } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { isPageBlock } from '../helpers';
+
+export const name = 'amp-story-page-remove';
+
+export const icon = 'trash';
+
+/**
+ * Renders remove page button.
+ *
+ * @return {ReactElement} The rendered trash menu item.
+ */
+const RemovePageSetting = () => {
+	const { removeBlock } = useDispatch( 'core/block-editor' );
+	const currentPageId = useSelect( ( select ) => select( 'amp/story' ).getCurrentPage(), [] );
+
+	const removePage = () => {
+		if ( ! isPageBlock( currentPageId ) ) {
+			return;
+		}
+		removeBlock( currentPageId );
+	};
+
+	return (
+		<PluginBlockSettingsMenuItem
+			icon="trash"
+			label={ __( 'Remove page', 'amp' ) }
+			role="menuitem"
+			onClick={ removePage }
+		/>
+	);
+};
+
+export const render = () => {
```

Comment:
This is similar to simply writing:

```
export const render = () => <RemovePageSetting />;
```

Reply:
Good spot. It was like after a refactor. Changed in https://github.com/ampproject/amp-wp/pull/3538/commits/5a65afcebe406c461f7fc134877f6b101f5d08da